### PR TITLE
feat: add new profiles

### PR DESCRIPTION
**Pull request by dabo505**
Two new profiles and some adjustments to OFCOM. Please merge if everything looks good...

### CODE REVIEW

**Review by tpluscode — APPROVED** [review]
This time `https://transport.ld.admin.ch/` has an invalid certificate. `health` domain works fine. 

Looks good otherwise.

**Comment by dabo505** [comment]
> This time `https://transport.ld.admin.ch/` has an invalid certificate. `health` domain works fine.
> 
> Looks good otherwise.

Thanks for the hint, I forwarded it to BAR.

**Comment by dabo505** [comment]
> ObservedBy should be ld.admin.ch/office/ namespace.On 11 Apr 2023 16:01, David Bokel ***@***.***> wrote: This time https://transport.ld.admin.ch/ has an invalid certificate. health domain works fine. Looks good otherwise. Thanks for the hint, I forwarded it to BAR. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Yes, but it doesn't exist for BAKOM afaik.

**Comment by l00mi** [comment]
https://ld.admin.ch/office/VII.1.6

File: .husky/organizations.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import { turtle } from '@tpluscode/rdf-string'
```
Comment (tpluscode) [inline .husky/organizations.ts:1]: How did file get here? :) Please remove it and update `apis/core/bootstrap/organizations.ts`

**Review by tpluscode — CHANGES_REQUESTED** [review]

File: .husky/organizations.ts (hunk 1 above)
Comment (dabo505) [reply .husky/organizations.ts]: Hm could it be that husky did that somehow? Else: sorry! Should be fixed now...

**Review by tpluscode — APPROVED** [review]

**Comment by tpluscode** [comment]
Good to merge @l00mi @dabo505 ?

**Review by l00mi — APPROVED** [review]

**Pull request merged**